permissions/resource_permissions.go

```diff
@@ -175,7 +175,7 @@ func (a PermissionsAPI) Update(objectID string, objectACL AccessControlChangeLis
 			PermissionLevel: "CAN_MANAGE",
 		})
 	}
-	if strings.HasPrefix(objectID, "/jobs") || strings.HasPrefix(objectID, "/pipelines") {
+	if strings.HasPrefix(objectID, "/jobs") || strings.HasPrefix(objectID, "/pipelines") || strings.HasPrefix(objectID, "/sql/warehouses") {
```

Comment:
In line 196, if user doesn't specify new owner in ACL list, can we use existing owner/creator instead of current user? With current code, whenever a different user applying tf, we changes the owner.

Reply:
line 254 is within the `delete` function? and line 196 here is the `update` function?
So if a warehouse currently has permissions [ (user1, is_owner), (team1, can_manage) ] and we specify new permissions in tf as [ (team1, can_manage) ] do we use `delete` or `update` function here?

Reply:
you are correct, we only add the `CreatorName` as owner when deleting permissions, but replacing the owner with current user when creating/updating.

This was consistent with the behaviour of this resource for jobs & pipelines, and I don't think it matters significantly as tf apply is usually done with a single identity, e.g. an sp rather than a different user every time

@alexott @mgyucht what are your thoughts?

Reply:
We do have use cases that multiple engineers run tf apply, so not changing resource owner would be expected behavior, and it also works fine for sp use case right?